tests/sentry/tasks/test_data_export.py

```diff
@@ -69,3 +72,36 @@ def test_assemble_download_issue_by_tag(self):
         raw1, raw2 = sorted([raw1, raw2])
         assert raw1.startswith("bar,1,")
         assert raw2.startswith("bar2,2,")
+
+    @patch("sentry.models.ExportedData.email_failure")
+    def test_issue_by_tag_errors(self, emailer):
+        de1 = ExportedData.objects.create(
+            user=self.user,
+            organization=self.org,
+            query_type=2,
+            query_info={
+                "project_id": self.project.id + 1,
```

Comment:
@leeandher If test setup changes, there could definitely be a `project_id` that's one higher than `self.project.id`. I would query it with some nonsense that won't ever have a matching project. Same goes for the event ID below.

Reply:
Made them both `-1`